py-scripts/lf_base_interop_profile.py

```diff
@@ -1481,6 +1931,65 @@ def main():
 
     parser.add_argument('--help_summary', default=None, action="store_true",
                         help='Show summary of what this script does')
+    parser.add_argument('--device_list', default=None,
```

Comment:
Future work can move argparser definition to helper `parse_args()`, as is in other scripts (trims down the `main()` function)